projects/storefrontlib/shared/components/card/card.component.ts

```diff
@@ -22,7 +28,12 @@ export interface Card {
   title?: string;
   textBold?: string;
   text?: Array<string>;
-  paragraphs?: Array<{ title?: string; text?: Array<string> }>;
+  paragraphs?: Array<{
+    title?: string;
```

Comment:
is there any need to make the title and text optional? - if not you could make them mandatory - that would ease the coding below.